README.md

```diff
@@ -203,6 +203,14 @@ Meeting notes are held in a [Google doc](https://docs.google.com/document/d/18JV
 
 You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-erlang).
 
+### PHP SIG
+
+The PHP SIG meets weekly on [Wednesday 10:30 AM PDT](https://calendar.google.com/calendar/embed?src=google.com_346fplg7bemb44783pege5amj4%40group.calendar.google.com&ctz=America%2FLos_Angeles) via Zoom. Follow this link to [join the meeting](https://zoom.us/j/8203130519).
```

Comment:
nit: can you say "PT" instead of "PDT"? (no need to update for PST :)

Reply:
```suggestion
The PHP SIG meets weekly on [Wednesday 10:30 AM PT](https://calendar.google.com/calendar/embed?src=google.com_346fplg7bemb44783pege5amj4%40group.calendar.google.com) via Zoom. Follow this link to [join the meeting](https://zoom.us/j/8203130519).
```